### PR TITLE
netxng: add possibility to control log verbosity with NetXNG.Debug

### PR DESCRIPTION
**Pull request by gganis**
Addresses ROOT-9311. The verbosity level can also be changed with the environment variable
XRD_LOGLEVEL. The possible levels are: "Info", "Debug", "Dump" .

### CODE REVIEW

**Comment by gganis** [comment]
@phsft-bot build just on mac1013/native, centos7/gcc7

**Comment by dpiparo** [comment]
Hi @gganis and @Axel-Naumann : this looks good. Can this be merged?

File: net/netxng/src/TNetXNGFile.cxx [hunk 1]
```diff
@@ -132,6 +132,11 @@ TNetXNGFile::TNetXNGFile(const char *url,
 {
    using namespace XrdCl;
 
+   // Set the log level
+   TString val = gEnv->GetValue("NetXNG.Debug",     "");
+   if (val.IsNull()) val = gSystem->Getenv("XRD_LOGLEVEL");
```
Comment (Axel-Naumann) [inline net/netxng/src/TNetXNGFile.cxx]: I'd assume that an env var is overriding any setting in rootrc, not the inverse?

Comment (gganis) [reply net/netxng/src/TNetXNGFile.cxx]: Right. I have uploaded the change. Thanks, G

**Pull request merged**